tools/cldr-apps/src/main/java/org/unicode/cldr/web/DBUtils.java

```diff
@@ -378,38 +310,19 @@ public boolean tableExists(String table) {
         }
     }
 
-    private static String canonTableName(String table) {
-        return db_Derby ? table.toUpperCase() : table;
+    private static final String canonTableName(String table) {
+        return table;
     }
 
     public static boolean tableHasColumn(Connection conn, String table, String column) {
         final String canonTable = canonTableName(table);
         final String canonColumn = canonTableName(column);
-        try {
-            if (db_Derby) {
-                ResultSet rs;
-                DatabaseMetaData dbmd = conn.getMetaData();
-                rs = dbmd.getColumns(null, null, canonTable, canonColumn);
-                if (rs.next()) {
-                    rs.close();
-                    // logger.severe("column " + table +"."+column + " did exist.");
-                    return true;
-                } else {
-                    SurveyLog.debug("column " + table + "." + column + " did not exist.");
-                    return false;
-                }
-            } else {
-                return sqlCount(
-                                conn,
-                                "select count(*) from information_schema.COLUMNS where table_name=? and column_name=?",
-                                canonTable,
-                                canonColumn)
-                        > 0;
-            }
-        } catch (SQLException se) {
-            SurveyMain.busted("While looking for column '" + table + "." + column + "': ", se);
-            return false; // NOTREACHED
-        }
+        return sqlCount(
+                        conn,
+                        "select count(*) from information_schema.COLUMNS where table_name=? and column_name=?",
```

Comment:
ideally all caps for the sql reserved words -- out of scope for this ticket though